index.Rmd

```diff
@@ -304,101 +303,127 @@ ggplot(us_data, aes(x = med_income, y = med_prop_val)) +
 
 One common and easy way to format your `ggplot` visualization is to add additional layers to the plot object using the `+` symbol. 
```

Comment:
Maybe here we can say something commenting on the plot we made above in 6.1.2 just to give a little more motivation for the code that follows. Kind of like: Yes, we have plotted a scatterplot, but we aren't done yet -- there's more that we can do to improve the interpretability of the plot. For instance, most people won't know what "mother_tongue" means without more explanation etc...

Reply:
Addressed in 36c7989